src/sbtc-releases/sbtc-dev/get-started.md

```diff
@@ -0,0 +1,47 @@
+# Get Started with sBTC 0.1
+
+## Local setup
+### Requirements
+* install [clarinet](https://github.com/hirosystems/clarinet)
+* checkout main branch of repository [github.com/stacks-network/sbtc](https://github.com/stacks-network/sbtc)
+
+### Configuration
+1. Create configuration file `config.json` in a new folder `romeo/testing` with the following content:
+```
+{
+  "state_directory": "./state",
+  "contract": "../asset-contract/contracts/asset.clar",
+  "wif": "YOUR_WIF_CREDENTIALS",
+  "bitcoin_node_url": "http://devnet:devnet@localhost:8001/api",
+  "stacks_node_url": "http://localhost:3999",
+  "stacks_transaction_fee": 2000,
+  "bitcoin_transaction_fee": 2000,
+  "contract_name": "sbtc-alpha-romeo-testing"
+}
+```
+2. Create a folder `state` in `testing` folder
+3. Create an empty file `log.ndjson`
+### Launch devnet
+
+In a first console
```

Comment:
Just a couple of small suggestions to improve the clarity and wording of these instructions.

```suggestion
Open up your terminal and run the following commands. When running `clarinet integrate`, you will be prompted to overwrite the deployment script, press `N` to say no.
```